devel/gitweb/Portfile

```diff
@@ -0,0 +1,105 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           cargo 1.0
+
+github.setup        yoannfleurydev gitweb 0.3.2 v
+revision            0
+
+description         Open the current remote git repository in your browser
+
+long_description    ${description}
+
+categories          devel
+platforms           darwin
+maintainers         {gmail.com:smanojkarthick @manojkarthick} \
+                    openmaintainer
+license             MIT
+installs_libs       no
+
+checksums           ${distname}${extract.suffix} \
+                    rmd160  53734c5f3066cf0b9e18655790b871673cceea6f \
+                    sha256  9a8887fe063fd619a40c15c099f47fddcb82b73720e3e87ccf76499e296d3591 \
+                    size    5907269
+
+depends_build       port:pkgconfig
```

Comment:
Let's make one enhancements here: `depends_build` -> `depends_build-append`

`depends_build` specifies the list of software required to build this port, and since you're using the `cargo` portgroup, that list starts out with `cargo`.  But again specifying `depends_build` here, you overwrite that list.  So safer in general to use `depends_build-append`.

Though we do have Portfiles doing this exact same thing, and they do need to be fixed over time.

Reply:
I wasn't aware of that - Thank you! I've updated the PR.